dashboard/app/main.go

```diff
@@ -1126,12 +1142,21 @@ func loadManagers(c context.Context, accessLevel AccessLevel, ns, manager string
 		if now.Sub(mgr.LastAlive) > 6*time.Hour {
 			uptime = 0
 		}
+		// TODO: ideally we'd want to explicitly detect and mark cases, when
+		// the build is too new to have some coverage report. For now we just
+		// query the latest uploaded cover report for the manager.
+		coverURL := ""
+		if config.CoverPath != "" {
```

Comment:
How will we configure this? syz-ci's won't upload reports using the old scheme anymore, right? So does it make sense to have CoverPath set for any kernels?

Reply:
Adjusted the code -- now syz-ci accepts and handles `CoverPath` if no asset storage is configured